spec/php-spec.coffee

```diff
@@ -942,6 +942,76 @@ describe 'PHP grammar', ->
           expect(tokens[3][0]).toEqual value: 'JAVASCRIPT', scopes: ['text.html.php', 'meta.embedded.block.php', 'source.php', 'string.unquoted.nowdoc.php', 'meta.embedded.js', 'punctuation.section.embedded.end.php', 'keyword.operator.nowdoc.php']
           expect(tokens[3][1]).toEqual value: ';', scopes: ['text.html.php', 'meta.embedded.block.php', 'source.php', 'punctuation.terminator.expression.php']
 
+      it 'should tokenize a heredoc with embedded javascript correctly', ->
```

Comment:
This doesn't need to be a separate `it`.  It can be combined under the existing JS tests.